.github/workflows/vue.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     - uses: actions/checkout@v2
     - id: Build-Vue
       uses: xRealNeon/VuePagesAction@1.0.1
+      env:
+        VUE_APP_INDEXER_URL: ${{ secrets.INDEXER_URL }}
```

Comment:
Just use `https://mainnet-node.like.co` as public config?

Reply:
Because we haven't deployed yet and I think Edmond and Phoebe may need to use it.

Reply:
Add default indexer url in config.js